README.md

```diff
@@ -1 +1,38 @@
-# kf-api-dataservice
\ No newline at end of file
+kf-api-dataservice
+==================
+
+[![license](https://img.shields.io/github/license/kids-first/kf-api-dataservice.svg?style=for-the-badge)](https://github.com/kids-first/kf-api-dataservice/blob/master/LICENSE)
+
+The Kids First Data Service provides a REST API to the Kids First data.
+
+## Development
+
+Start a development service of your own:
+
```

Comment:
Minor change request, can we do something like: 

To start a development service of your own:

Clone the git repository via

```
git clone git@github.com:kids-first/kf-api-dataservice.git
```

Install dependencies

```
pip install -r requirements.txt
```

Set the FLASK_APP environment variable

```
export FLASK_APP=manage.py
```

Initialize the database
```
flask db init
flask db upgrade
```

Run the service

```
flask run
```